### ISSUE TITLE
F16 Display formatting issue, not present when in F/A-18 Mode

### ISSUE THREAD

**Issue by NWSpitfire** [jg-storey/ded#2]
Hi,
When trying this in F-16 mode the OLED correctly displays the last few lines of the DED home page on the Left Hand Side of the screen but also displays other odd characters and has one "digit" pop up and cycle through all characters every few seconds.

In LIST menu, the last 3 letters of the "E" and "0" menu are displayed correctly (LNK and ISC), but the rest of the menu is drawn off the end of the screen (I'm guessing?).

In F/A-18 mode all text is placed and formatted correctly and works well (the 2555555 was to test that the UFC scratchpad was working properly).

I checked my wiring and it matches the github wiring diagram. I am using an 328P Arduino Nano with the display (2.8" 256x64 being driven by an SSD1322) in 4 wire SPI mode.

I have tried with both (the reccomended) DCS-BIOS v0.10.0 and the newer DCS-Bios Fork (DCS Flight Panels one) both had the same problem in F16 mode but worked fine in F/A-18 Mode. 

I haven't ever coded OLED's so I'm not sure what to look for, but I havent modified your code in any way.

Do you have any idea what could be occurring? Have you come across this problem with your unit?

Thank you for publishing the code, if you need any other troubleshooting or information let me know. I have attached some pictures to help also.

DCS Version: V2.8.6.41363
DCS BIOS Version: 0.10.0
DCS BIOS Fork Version: 2.8.4

![IMG_4839](https://github.com/jg-storey/ded/assets/45697172/5cb21499-7b0b-4725-82f9-6b701590c924)

![IMG_4840](https://github.com/jg-storey/ded/assets/45697172/7fee9b35-31c5-4b5c-b763-07d35de19b99)

**Comment by jg-storey** [comment]
@NWSpitfire Thanks for the feedback. I guess DCS must have changed some of the export settings for the Viper.  Late last year I changed over to the DCS Flight Panels fork because it exports all of the Viper's DED pages. If you want to see my code I've uploaded it to 
https://github.com/jg-storey/flight-panels-ded
but note that I'm providing it "as is", meaning I haven't cleaned it up for general consumption. I also don't know if it's compatible with the latest Flight Panels DCS-bios release.
